src/h1/mod.rs

```diff
@@ -40,6 +43,8 @@ pub struct H1Client {
     #[cfg(any(feature = "native-tls", feature = "rustls"))]
     https_pools: HttpsPool,
     max_concurrent_connections: usize,
+    // #[cfg(feature = "unstable-config")]
+    config: Option<Config>,
```

Comment:
Why not just have a Config::default() by default, instead of an `Option<Config>`?

Reply:
Space savings, I suppose? I guess it doesn't matter much either way.